Files/Views/SettingsPages/FilesAndFolders.xaml

```diff
@@ -71,6 +71,12 @@
                     Header="Enable individual preferences for individual directories"
                     HeaderTemplate="{StaticResource CustomHeaderStyle}"
                     IsOn="{Binding AreLayoutPreferencesPerFolder, Mode=TwoWay}" />
+
+                <ToggleSwitch
+                    x:Uid="SettingsEnableAdaptiveLayout"
+                    Header="Enable Adaptive Layout that dynamically adjusts based on the directory context"
```

Comment:
The text here is a bit long, do you have any ideas how it can be simplfied?

Reply:
I'll think of that. Honestly, #2905 would be perfect fit here